src/mon/FSCommands.h

```diff
@@ -47,6 +47,7 @@ class FileSystemCommandHandler : protected CommandHandler
       const int64_t pool_id,
       int type,
       bool force,
+      bool allow_overlay,
```

Comment:
IMO, you can reuse the `force` parameter here and they seems having the same logic.

And set the `force = force || allow_overlay` ? Maybe you also should rename the `force` to a better name, such as `allow_overlay` ?

Reply:
force is used for `--force` while allow_overlay is used for `--allow_dangerous_metadata_overlay`.
Hence keeping it separately to avoid confusion.